src/physics/cam/clubb_intr.F90

```diff
@@ -2942,6 +2973,11 @@ subroutine clubb_tend_cam( &
       rtp2_zt = zm2zt_api(rtp2_in)
       thl2_zt = zm2zt_api(thlp2_in)
       wp2_zt  = zm2zt_api(wp2_in)
+!+++MKW
+      ! Need moist_qc and cloudfrac on thermo grid for output
+      mf_qc_zt = zm2zt_api(mf_moist_qc)

```

Comment:
and just as a double check, this mf_moist_qc is an in cloud mixing ratio, not a grid mean, right?

Reply:
Correct: in-cloud MR